setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = seed_isort_config
-version = 1.9.1
+version = 1.9.2
```

Comment:
I'll bump the version later -- could you rebase this out please

Reply:
Done.